tests/common/testUtils.ts

```diff
@@ -80,13 +80,13 @@ export function ts2pyAsync(f: string): Promise<string> {
             let program = pxtc.getTSProgram(opts);
             // TODO: if needed, we can re-use the CallInfo annotations the blockly decompiler can add
             // annotate(program, tsFile, target || (pxt.appTarget && pxt.appTarget.compile));
-            const decompiled = (pxt as any).py.decompileToPythonHelper(program, "main.ts");
+            const decompiled = (pxt as any).py.decompileToPython(program, "main.ts");
```

Comment:
Can drop the ``pxt as any`` here, right?

Reply:
I'm not going to change it since it's not a part of this PR